app/state/parse.js

```diff
@@ -29,14 +29,14 @@ export const reducer = (previousState = initialState, action) => {
           );
         }
         return initialState
-          .set("errors", Immutable.List())
+          .set("error", Immutable.List())
```

Comment:
This line is not really needed, as we are using the `initialState` here to reset the state, so `error` is already an empty list.